docs/index.md

```diff
@@ -1063,6 +1064,12 @@ tcp://192.168.99.109:2376
 #### Amazon Web Services
 Create machines on [Amazon Web Services](http://aws.amazon.com).  You will need an Access Key ID, Secret Access Key and a VPC ID.  To find the VPC ID, login to the AWS console and go to Services -> VPC -> Your VPCs.  Select the one where you would like to launch the instance.
 
+Use `amazonec2` for the `--driver` and include the 3 required parameters. Depending upon the configured VPC, you may have to specify `--amazonec2-zone` if a subnet does not exist in Availability Zone `a`.
```

Comment:
Thank you for the contribution.  Would you be willing to clean this section up a bit so it is easier to read.  I had one question -- I don't understand the reference to Availability Zone 'a' --- can you clarify.  Is Availability Zone 'a' the default and if it isn't in the default user needs to supply a parameter that points to the VPC's zone?

---> https://gist.github.com/moxiegirl/a7fe776b7c4127f97fd7

#### Amazon Web Services

To create machines on [Amazon Web Services](http://aws.amazon.com) (AWS), you must supply three required parameters:

* Access Key ID
* Secret Access Key
* VPC ID

You obtain these values from AWS.  To find the VPC ID:

!. Login to the AWS console 
2. Go to **Services -> VPC -> Your VPCs**.  
3. Select the VPC where you would like to launch the machine instance.

To create the machien instance, specify `--driver amazonec2` and the three required parameters. 

```
$ docker-machine create --driver amazonec2 --amazonec2-access-key AKI******* --amazonec2-secret-key 8T93C********* --amazonec2-vpc-id vpc-****** aws01
```

Depending upon the VPC you choose, you may also have to specify `--amazonec2-zone` if a subnet does not exist in **Availability Zone** `a`.